tests/process_monitor/test_process_monitor.py

```diff
@@ -627,7 +627,7 @@ def test_MantarrayProcessesMonitor__correctly_sets_system_status_to_live_view_ac
     assert shared_values_dict["system_status"] == RECORDING_STATE
 
 
-def test_MantarrayProcessesMonitor__sets_system_status_to_server_ready_after_subprocesses_finish_start_up(
+def test_MantarrayProcessesMonitor__sets_system_status_to_websocket_ready_after_subprocesses_finish_start_up(
```

Comment:
This test name should be test_MantarrayProcessesMonitor__sets_system_status_to_server_ready_after_subprocesses_finish_start_up__and_websocket_connection_is_made